client/backend_arguments.py

```diff
@@ -69,7 +69,8 @@ def serialize(self) -> Dict[str, object]:
         }
 
     def get_checked_directory_allowlist(self) -> Set[str]:
-        return {element.path() for element in self.elements}
+        # pyre-fixme: Incompatible return type [7]: Expected Set[str] but got Set[Optional[str]].
+        return {element.path() for element in self.elements if element is not None}
```

Comment:
element's path [can be empty](https://fburl.com/cz3a6nu7). you should also check that it's not `None`